rider/.idea/encodings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we really need this file?

Reply:
Possibly not. But I was fed up with it in my "unversioned files". Looking at the file, and the options page that seems to drive it, it looks like it's something that should be shared. And since IntelliJ has no sensible scheme for knowing if a file should be committed or not, I flipped a coin and it ended in.